zigzag/data/junit.xsd

```diff
@@ -0,0 +1,151 @@
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element ref="property" minOccurs="0" maxOccurs="unbounded"/>
+            </xs:sequence>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:element name="property">
+        <xs:complexType>
+            <xs:attribute name="name" type="xs:normalizedString" use="required"/>
+            <xs:attribute name="value" type="xs:normalizedString" use="required"/>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:element name="system-err" type="xs:normalizedString"/>
+    <xs:element name="system-out" type="xs:normalizedString"/>
+    <xs:element name="rerunFailure" type="rerunType"/>
+    <xs:element name="rerunError" type="rerunType"/>
+    <xs:element name="flakyFailure" type="rerunType"/>
+    <xs:element name="flakyError" type="rerunType"/>
+
+    <xs:element name="testcase">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:choice minOccurs="0" maxOccurs="unbounded">
+                    <xs:element ref="skipped"/>
+                    <xs:element ref="error"/>
+                    <xs:element ref="failure"/>
+                    <xs:element ref="rerunFailure" minOccurs="0" maxOccurs="unbounded"/>
+                    <xs:element ref="rerunError" minOccurs="0" maxOccurs="unbounded"/>
+                    <xs:element ref="flakyFailure" minOccurs="0" maxOccurs="unbounded"/>
+                    <xs:element ref="flakyError" minOccurs="0" maxOccurs="unbounded"/>
+                    <xs:element ref="system-out"/>
+                    <xs:element ref="system-err"/>
+                    <xs:element ref="properties"/>
+                </xs:choice>
+            </xs:sequence>
+            <xs:attribute name="name" type="xs:normalizedString" use="required"/>
+            <xs:attribute name="time" type="xs:normalizedString"/>
+            <xs:attribute name="classname" type="xs:normalizedString"/>
+            <xs:attribute name="group" type="xs:normalizedString"/>
+            <xs:attribute name="file" type="xs:normalizedString"/>
+            <xs:attribute name="line" type="xs:normalizedString"/>
+        </xs:complexType>
+    </xs:element>
+
+    <xs:element name="testsuite">
+        <xs:complexType>
+            <xs:choice minOccurs="0" maxOccurs="unbounded">
+                <xs:element ref="testsuite"/>
+                <xs:element ref="properties"/>
+                <xs:element ref="testcase"/>
+                <xs:element ref="system-out"/>
+                <xs:element ref="system-err"/>
+            </xs:choice>
+            <xs:attribute name="name" type="xs:normalizedString" use="required"/>
+            <xs:attribute name="tests" type="xs:normalizedString" use="required"/>
+            <xs:attribute name="failures" type="xs:normalizedString" use="required"/>
+            <xs:attribute name="errors" type="xs:normalizedString" use="required"/>
+            <xs:attribute name="skips" type="xs:normalizedString"/>
```

Comment:
I would add a comment letting people know that this was specifically added for pytest behavior.